src/main/webapp/js/projects/react/components/MentorCard.js

```diff
@@ -0,0 +1,38 @@
+import {mentorType} from '../prop_types.js';
+import TagList from './TagList.js';
+
+/**
+ * A card to display basic information about a project mentor.
+ * @param {Object} props
+ * @return {React.Component} Returns the react component.
+ */
+export default function MentorCard(props) {
+  const mentor = props.mentor;
+
+  return (
+    <div className="card project-card h-100">
+      <div className="d-flex flex-column card-body">
+        <div className="d-flex justify-content-center mb-2">
+          <img className="mentor-picture" src={mentor.image} alt="" />
```

Comment:
nit: provide an `alt`? Good for accessibility.

Reply:
That's a good point. I initially kept it blank as is convention when an image is considered "decorative," but now that I think about it it seems like it might be important for context. Will add